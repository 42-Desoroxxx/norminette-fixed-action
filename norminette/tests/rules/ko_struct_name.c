typedef struct t_toto s_toto;
union u_toto u_var;
int s_int;

typedef struct toto {
    struct v_toto;
    union s_toto;
    typedef union test s_test;
    enum g_toto;
}   u_struct;

int main(void) {
    return (0);
}