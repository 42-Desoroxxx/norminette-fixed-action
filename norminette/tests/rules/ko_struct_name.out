tests/rules/ko_struct_name.c: KO!
	USER_DEFINED_TYPEDEF (line: 1, col: 25):	User defined typedef must start with t_
	STRUCT_TYPE_NAMING (line: 1, col: 25):	Structure name must start with s_
	GLOBAL_VAR_NAMING (line: 3, col: 25):	Global variable must start with g_
	STRUCT_TYPE_NAMING (line: 6, col: 12):	Structure name must start with s_
	UNION_TYPE_NAMING (line: 7, col: 11):	Union name must start with u_
	USER_DEFINED_TYPEDEF (line: 8, col: 25):	User defined typedef must start with t_
	UNION_TYPE_NAMING (line: 8, col: 25):	Union name must start with u_
	ENUM_TYPE_NAMING (line: 9, col: 10):	Enum name must start with e_
	USER_DEFINED_TYPEDEF (line: 10, col: 5):	User defined typedef must start with t_
	STRUCT_TYPE_NAMING (line: 10, col: 5):	Structure name must start with s_
